glossary.rst

```diff
@@ -5,34 +5,58 @@ Glossary
 Ansible
 *******
 
-Ansible is software that automates software provisioning, configuration management, and application deployment.
+`Ansible <https://www.ansible.com/>`_ is software that automates software provisioning, configuration management, and application deployment.
 
 *****************
-ansible openwisp2
+ansible-openwisp2
 *****************
 
-Ansible role that installs the openwisp2 controller.
+Ansible role that installs the `openwisp2 <https://github.com/openwisp/ansible-openwisp2/>`_ controller.
+
+********************
+django-netjsonconfig
+********************
+
+Configuration manager for embedded devices, implemented as a reusable django-app
```

Comment:
please add a final dot (.) and a link to the django-netjsonconfig repository, it would be good if you could specify that this is one of the core component of openwisp2, the web interface through which configurations are manipulated

Reply:
Okay, will do it.